src/Microsoft.AspNetCore.Mvc.RazorPages/Filters/PageHandlerPageFilter.cs

```diff
@@ -3,9 +3,9 @@
 
 using System;
 using System.Threading.Tasks;
-using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.AspNetCore.Mvc.RazorPages;
 
-namespace Microsoft.AspNetCore.Mvc.RazorPages.Internal
+namespace Microsoft.AspNetCore.Mvc.Filters
 {
     public class PageHandlerPageFilter : IAsyncPageFilter, IOrderedFilter
```

Comment:
Is this included in our upcoming review of newly `public` types?

Reply:
Missed updating this.